__tests__/app.test.js

```diff
@@ -32,4 +32,28 @@ it('responds with 404 error if not path is found', ()=>{
     })
 });
   })
+  describe('/api/articles/:article_id', ()=>{
+    test('responds with an article object', ()=>{
+      return request(app).get('/api/articles/1').expect(200).then((response)=>{
+         expect(response.body.article).toHaveLength(1)
+        response.body.article.forEach((article) =>{
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number)
+            }))
+        })
+      })
+    })
+    test('responds with 400 for bad requests', ()=>{
```

Comment:
What about error handling if someone puts a valid ID (a number) but no article ID of that number exists in the db?